KsApi/models/Config.swift

```diff
@@ -1,6 +1,7 @@
 public enum Experiment {
   public enum Name: String {
     case creatorsNameDiscovery = "show_created_by_discovery"
+    case nativeCheckoutV1 = "native_checkout_v1"
```

Comment:
Just wondering about the naming of this ... is this named `native` after our team (hence it's a native team feature) or is this named after the feature `native checkout`? 

If it's the first then I think it's fine. 
If it's the later then V1 is technically `web checkout` so not sure if it makes sense but don't feel strongly about it (just wanted to bring this up).

Reply:
This name starts with `native` so it becomes clear on the Kickstarter app that it's a native experiment (so no one deletes it by accident).